gtk-4.yaml

```diff
@@ -0,0 +1,149 @@
+package:
+  name: gtk-4
+  version: 4.15.5
+  epoch: 0
+  description: The GTK+ Toolkit (v4)
+  copyright:
+    - license: LGPL-2.1-or-later
+
+environment:
+  contents:
+    packages:
+      - at-spi2-core-dev
+      - autoconf
+      - automake
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - cairo-dev
+      - cmake
+      - cups-dev
+      - dbus-dev
+      - expat-dev
+      - fontconfig-dev
+      - fribidi-dev
+      - gdk-pixbuf-dev
+      - gettext-dev
+      - gi-docgen
+      - git
+      - glib-dev
+      - glib-gir
+      - gobject-introspection-dev
+      - graphene-dev
+      - gst-plugins-bad-dev
+      - gst-plugins-base-dev
+      - gstreamer-dev
+      - gtk-doc
+      - harfbuzz-dev
+      - hicolor-icon-theme
+      - iso-codes-dev
+      - libatk-bridge-2.0
+      - libdrm-dev
+      - libepoxy-dev
+      - libice-dev
+      - libtool
+      - libx11-dev
+      - libxcomposite-dev
+      - libxcursor-dev
+      - libxdamage-dev
+      - libxext-dev
+      - libxfixes-dev
+      - libxft-dev
+      - libxi-dev
+      - libxinerama-dev
+      - libxkbcommon
+      - libxkbcommon-dev
+      - libxrandr-dev
+      - libxtst-dev
+      - mesa-dev
+      - meson
+      - ninja
+      - orc-compiler
+      - orc-dev
+      - pango-dev
+      - py3-docutils
+      - py3-gi-docgen
+      - py3-packaging
+      - sassc
+      - shaderc
+      - shared-mime-info
+      - smartypants
+      - spirv-headers
+      # TODO: right now shaderc also inclused spirv-tools, causin a conflict need to fix that,
```

Comment:
Not sure i understand this comment - mentions we use our own spirv-tools, but then we've it commented out below, is this still valid?

Reply:
Yes, I have created a new package called shaderc which now also include the spirv-tools within it. I will create a Issue to track this and make sure we can use shaderc with our spirv-tools.

Reply:
https://github.com/wolfi-dev/os/pull/26125/files#diff-95912073ffd0aac4ceb05bf2c6ee2eb41fa14676a714207ad5f9970ccb2ac557R25 It needs to be done.